libtransmission/utils.cc

```diff
@@ -555,11 +555,13 @@ std::string tr_strratio(double ratio, char const* infinity)
 
 // ---
 
-bool tr_file_move(std::string_view oldpath_in, std::string_view newpath_in, tr_error* error)
+namespace
+{
+namespace tr_file_move_impl
 {
-    auto const oldpath = tr_pathbuf{ oldpath_in };
-    auto const newpath = tr_pathbuf{ newpath_in };
 
+bool pathChecks(std::string_view& oldpath, std::string_view& newpath, tr_error* error)
```

Comment:
- please use `snake_case` for new code
- please [pass string_view by value](https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/)

```suggestion
bool check_paths(std::string_view const oldpath, std::string_view const newpath, tr_error* error)
```

Reply:
Fixed.